arduino_unit_tests/cc1120_regs.h

```diff
@@ -0,0 +1,75 @@
+#ifndef REGS_CC1120_H
+#define REGS_CC1120_H
+
+#define IOCFG3              0x00
```

Comment:
I'd add the part name to each of these registers. So instead of IOCFG3, I'd use CC1120_IOCFG3.

Reply:
1. That sounds like a good idea for when we integrate with the CDH code, I'll make that change.
2. I'm not sure what the best practice is either, I don't see a reason why we can't integrate it in, but I want to figure out what the best approach is before I do that